resources/qml/Dialogs/AskOpenAsProjectOrModelsDialog.qml

```diff
@@ -97,29 +97,32 @@ UM.Dialog
             text: catalog.i18nc("@text:window", "Remember my choice")
             checked: UM.Preferences.getValue("cura/choice_on_open_project") != "always_ask"
         }
+    }
 
-        // Buttons
-        Item {
-            id: buttonBar
-            anchors.right: parent.right
-            anchors.left: parent.left
-            height: childrenRect.height
-
-            Button {
-                id: openAsProjectButton
-                text: catalog.i18nc("@action:button", "Open as project")
-                anchors.right: importModelsButton.left
-                anchors.rightMargin: UM.Theme.getSize("default_margin").width
-                isDefault: true
-                onClicked: loadProjectFile()
-            }
-
-            Button {
-                id: importModelsButton
-                text: catalog.i18nc("@action:button", "Import models")
-                anchors.right: parent.right
-                onClicked: loadModelFiles()
-            }
+    Item
```

Comment:
Is it really needed to put an item around button group?

Reply:
If I set it to on the root of the element I get the following error
```
2022-01-25 16:12:52,495 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [428]: /Cura/resources/qml/Menus/RecentFilesMenu.qml:36:5: Type Cura.AskOpenAsProjectOrModelsDialog unavailable
2022-01-25 16:12:52,495 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [428]: /Cura/cura/../resources/qml/Dialogs/AskOpenAsProjectOrModelsDialog.qml:103:5: Cannot assign object to list property "contents"
```

Reply:
I'm not sure if this is needed though. This was mainly to highlight the default button choice. The styling is yet to be implemented. These buttons will probably change to `Cura.PrimaryButton` and `Cura.SecondaryButton`, making the use of the `ButtonGroup` redundant.